docs/source/oauth2_priv.rst

```diff
@@ -0,0 +1,126 @@
+Privacy-Enhanced OAuth 2.0
+==========================
+
+**Status:** DRAFT
+
+.. contents:: Table of Contents
+    :local:
+    :backlinks: top
+
+Contributors
+------------
+
+* George Aristy (SecureKey Techologies)
+
+Introduction
+------------
+
+Purpose of this document
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+This document describes a reference implementation of OAuth 2.0 with
+unregistered clients communicating and authenticating securely over the
+backchannel with decentralized identifiers and verifiable credentials.
+
+Motivation
+^^^^^^^^^^
+
+There is a desire to leverage existing OAuth 2.0 infrastructure to build a
+privacy-enhanced data sharing solution.
+
+Finalized in 2012, OAuth 2.0 ([RFC6749]_), is an established authorization
+framework well suited to give a piece of software access to protected
+resources with the owner's consent. It was not, however, designed with the
+principles of Privacy by Design ([PRIV-DESIGN]_) in mind.
+
+First published in 2009, the principles of privacy by design became widely
+known after the GDPR adopted them ([GDPR-PRIV]_) and began enforcement in
+2018. We seek to address two key principles of privacy by design that OAuth
+2.0 does not:
```

Comment:
OAuth 2.0, but in the context of this scenario.

Reply:
Sorry, I don't understand.